hooks/useLobbyActions.ts

```diff
@@ -44,8 +44,19 @@ export default function useLobbyActions() {
   }
 
   function onStartGame() {
-    const status: RoomStatus = "waiting";
-    update(roomRef, { status });
+    runTransaction(teamsRef, (teams: Team[]) => {
+      if (teams.length < 2) canBeStarted = false;
+      teams.forEach((team, index) => {
+        if (index != 0 && team.members?.length < 2) canBeStarted = false;
```

Comment:
This does not account for the case where all teams are empty. It's a little gotcha in javascript, more details in today's meeting.

Reply:
@seannimmo @jg3835 `team.members?.length < 2` is true `members` is undefined which what happens when you fetch an empty array from firebase. 
An easy fix it adding `team.members ??= []` in the line before.